src/ScriptCs.Hosting/ScriptServicesBuilder.cs

```diff
@@ -83,14 +85,14 @@ public IScriptServicesBuilder LoadModules(string extension, params string[] modu
                 ? "mono"
                 : "roslyn";
 
-            moduleNames = moduleNames.Union(new[] {engineModule}).ToArray();
+            moduleNames = moduleNames.Union(new[] { engineModule }).ToArray();
 
             var config = new ModuleConfiguration(_cache, _scriptName, _repl, _logLevel, _debug, Overrides);
             var loader = InitializationServices.GetModuleLoader();
 
             var fs = InitializationServices.GetFileSystem();
 
-            var folders = new[] {fs.GlobalFolder};
+            var folders = new[] { fs.GlobalFolder };
```

Comment:
I figured that was R#.....

On Mon, Apr 6, 2015 at 9:22 PM Christopher Bennage <notifications@github.com>
wrote:

> In src/ScriptCs.Hosting/ScriptServicesBuilder.cs
> <https://github.com/scriptcs/scriptcs/pull/1016#discussion_r27850746>:
>
> >
> >              var config = new ModuleConfiguration(_cache, _scriptName, _repl, _logLevel, _debug, Overrides);
> >              var loader = InitializationServices.GetModuleLoader();
> >
> >              var fs = InitializationServices.GetFileSystem();
> >
> > -            var folders = new[] {fs.GlobalFolder};
> > +            var folders = new[] { fs.GlobalFolder };
>
> Sorry, this was R#. [image: :confounded:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/1016/files#r27850746>.
>